### DISCUSSION TITLE
Control of entities from the bridge

### DISCUSSION THREAD

**Discussion by DJBenson** [t0bst4r/home-assistant-matter-hub#105] [General]
Is there a plan to allow entities to be controlled from the bridge at some point? When you open the bridge and get all the entities laid out like that, it's fairly obvious to try and click on them but currently they are just representing state - just wondered if it was ever on the roadmap to allow control from the bridge.

**Comment by t0bst4r** [comment]
For now it is not planned. you already have HA and your voice assistant to control devices. Why should we need it here? 😅

**Reply by DJBenson** [reply]
I didn't say I needed it - I asked if it was on the roadmap. You could argue the same about Homebridge but that allows control from the bridge so there was clearly a use case for it.

**Reply by t0bst4r** [reply]
Okay, got it! 👍🏼
